### DISCUSSION TITLE
Nested collections?

### DISCUSSION THREAD

**Discussion by aam1r** [outline/outline#1723] [General]
I like how documents can be nested and was wondering if it's possible for Collections to be nested? My use cases are below

**1. Hierarchical Granular Permissions**

Consider the following example scenario:

```
🗂 Engineering
  🗂 Backend
    📄 Code Style
    📄 User Guide
  🗂 Mobile
    🗂 Android 
      📄 Java Getting Started Guide
    🗂 iOS
      📄 Apple Build Process
      📄 Apple Signing Keys
      📄 SwiftUI 101
🗂 Finance
🗂 HR
```

It would be really nice to only share the "Android" folder with the Android team, "iOS" with the iOS team, and so on.

**2. Better Organization**

Having nested collections allows for better organizing as it's more clear what's a "folder" and what the sub-folders are.

**Comment by aam1r** [comment]
**3. Imports**

It seems like I can only import documents into a Collection, and not to a nested Document.

It's a bit annoying to import documents into a Collection and then move them around. It would be awesome if I could just import documents directly into a nested collection.

**Reply by tommoor** [reply]
You can import into a nested document by dragging onto it in the sidebar, we should add a menu item for this also.

Edit: Created an issue here: https://github.com/outline/outline/issues/1940

**Comment by chrisshaw** [comment]
I have upvoted this as well as https://github.com/outline/outline/discussions/2133. My use cases are

## 1:1s meeting notes

These can sometimes be sensitive and should be scoped to only the single person I am meeting with.

Right now, it looks like my option is creating a giant, flat list of private collections for each combination of people I want to hold sensitive conversations with.

I might add that these are the documents that are easy to put into Outline as a way to get the team on board with using it more. But right now it's not suitable for them.

## Continuous improvement

I'd like to include this as a nested document set underneath a team Collection, but this should be private to the team, while the rest is public to the rest of the organization. (While one could argue we should share these, often these experiments involve team happiness, and at least for now keeping it internal preserves psychological safety.)

**Reply by tommoor** [reply]
1:1's is a great usecase that we'd love to support. There is another issue here ([individual document sharing](https://github.com/outline/outline/issues/2359)) that would also potentially handle this type of workflow.
